meta-refkit/recipes-devtools/nodejs/nodejs_4.5.0.bbappend

```diff
@@ -0,0 +1,17 @@
+# NodeJS needs Python for node-gyp which is a NPM module bundled with
+# GYP together. GYP is a build system generator written in Python that
+# stands for "Generate Your Projects" and is used for building Node
+# modules written in C. Since Refkit doesn't support on-target
```

Comment:
Submitted https://github.com/ostroproject/meta-iot-web/pull/37 introducing nodejs-gyp, but then the supported recipes check fails even though nodejs-gyp is not installed. It needs to be amended to ignore RDEPENDS for not installed packages.